### DISCUSSION TITLE
Improve package quality by addressing issues reported on socket.dev

### DISCUSSION THREAD

**Discussion by josdejong** [josdejong/mathjs#3013] [Ideas]
There are probably some issues that are easy to resolve. socket.dev gives a clear insight in some (possible) issues:

https://socket.dev/npm/package/mathjs/overview

**Comment by gwhitney** [comment]
I clicked on the link but didn't see any issues listed there?

**Reply by josdejong** [reply]
Hm you're right, something changed apparently. This is not actionable anymore, let's close this discussion.
